### ISSUE TITLE
BUG: cron service 

### ISSUE THREAD

**Issue by osnet** [mlan/docker-kopano#16]
the settings for the internal docker cron are not overwritten  with force_config instead they get appended

i checked  with 

cat 
root@96f2dcc77ba7:/# cat /etc/cron.d/docker-crontab

```
0 1 * * * root kopano-archiver -A
0 3 * * 0 root kopano-archiver -C
0 1 * * * root kopano-archiver -A
0 3 * * 0 root kopano-archiver -C
0 1 * * * root kopano-archiver -A
0 3 * * 0 root kopano-archiver -C
0 1 * * * root kopano-archiver -A
0 3 * * 0 root kopano-archiver -C
0 1 * * * root /usr/sbin/kopano-archiver -A
0 3 * * 0 root /usr/sbin/kopano-archiver -C
*/2 * * * * root /usr/sbin/kopano-backup -O /backups
0 1 * * * root /usr/sbin/kopano-archiver -A
0 3 * * 0 root /usr/sbin/kopano-archiver -C
30 4 * * * root /usr/sbin/kopano-backup -O /backups
```

i only took notice of that cause my .env file stats  kopano backup at 04:30  and i was wondering why still everey 2 mins the jo starts

please fix : ) thanks

**Comment by mlan** [comment]
@osnet 

I think I know what went wrong. I will make sure that the `CRONTAB_ENTRY` are more robust.

In the mean time there are a couple of options that can fix this. You can choose any one from

1. run the `mlan/kopano` image without mounting a volume at `/etc/kopano`
2. run the `mlan/kopano` image with a empty/virgin volume mounted at `/etc/kopano`
3. copy the [`docker-crontab`](https://raw.githubusercontent.com/mlan/docker-kopano/master/src/docker/config/docker-crontab) file to `/etc/kopano` inside your container. that is; `cd /etc/kopano && wget https://raw.githubusercontent.com/mlan/docker-kopano/master/src/docker/config/docker-crontab`

**Comment by osnet** [comment]
ja, no stress mate ... 
even editing ``/etc/cron.d/docker-cron``  affects the live environment

**Comment by mlan** [comment]
Hi @osnet 

I have released f82c64cd477e65a9622c75e79e591f3ec99d3dc4 which I hope will fix your issue. Please find some details here [crontab](https://github.com/mlan/docker-kopano#crontab).
